### ISSUE TITLE
Is there a plan to join the feature of obtaining the list of installed software in the system?

### ISSUE THREAD

**Issue by jincandev** [sebhildebrandt/systeminformation#811]

**Comment by sebhildebrandt** [comment]
@jincandev yes! Version 6 of then package (which I am working on) will have this feature.

**Comment by shoustech** [comment]
@sebhildebrandt I had to do this for my app and found that I got the best data from the following commands:

`reg query HKLM\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall /s`
`reg query HKLM\Software\Microsoft\Windows\CurrentVersion\Uninstall /s`

I was also using WMI but found that the reg queries had all of the information that WMI did and then some.

I could probably put together a PR for the current version 5 if you are interested.

**Comment by sebhildebrandt** [comment]
@shoustech … for macOS and Windows I have a pretty good solution already, for Linux this is a little more tricky as there are different package managers, snap, … in place ;-) But yes, I could also add this code to version 5 … as soon as I have time for it, I will have a look on it.

**Comment by sebhildebrandt** [comment]
@jincandev @shoustech this is now implemented in Version 6 (currently out as a beta version). Please refer to the docs https://systeminformation.io/v6/. Would be happy if you can test it. Closing it for now.
